CONTRIBUTING.md

```diff
@@ -86,6 +86,62 @@ You can then fill in the `term`, `def`, and (if appropriate) `acronym` for that
       toutes les fonctions.
 ```
 
+## 3. Adding a new language
```

Comment:
I added the numbering here because the previous sections were numbered, however, they don't need to be sequential and given the subheadings below, I think it would be best to remove the sequential numbers from the 2nd-level headings.